### PR TITLE
Add sentry integration

### PR DESCRIPTION
**Pull request by hb0**
Pre-Reviewed, looks all good.

Wait for feedback in Sentry forum before finalizing terms and bumping terms version.

### CODE REVIEW

**Review by muthenberg — CHANGES_REQUESTED** [review]
One small comment

File: measuring-client/build.gradle [hunk 1]
```diff
@@ -56,7 +56,7 @@ android {
 
     // Load credentials
     def properties = new Properties()
-    properties.load(new FileInputStream("local.properties"))
+    file("local.properties").withInputStream { properties.load(it) }
```
Comment (hb0) [inline measuring-client/build.gradle:59]: This groovy method should auto-close the stream, see it's [documentation](http://docs.groovy-lang.org/latest/html/groovy-jdk/java/io/File.html):
"Create a new InputStream for this file and passes it into the closure. This method ensures the stream is closed after the closure returns."

File: measuring-client/build.gradle [hunk 2]
```diff
@@ -43,6 +54,10 @@ android {
         return stdout.toString().trim()
     }
 
+    // Load credentials
+    def properties = new Properties()
+    properties.load(new FileInputStream("local.properties"))
```
[the remark this answers is not in the document]
Comment (hb0) [reply measuring-client/build.gradle]: See below

File: measuring-client/build.gradle (hunk 1 above)
Comment (hb0) [reply measuring-client/build.gradle]: Didn't find or see anything else used in Gradle.

**Review by muthenberg — APPROVED** [review]

**Pull request merged**